core/storage/suggestion/gae_models.py

```diff
@@ -157,6 +157,9 @@ class GeneralSuggestionModel(base_models.BaseModel):
     # The ISO 639-1 code used to query suggestions by language, or None if the
     # suggestion type is not queryable by language.
     language_code = datastore_services.StringProperty(indexed=True)
+    # The flag that will suggest whether the suggestion is edited by the
```

Comment:
The flag doesn't "suggest" anything -- maybe you meant to write "A flag that indicates whether ..."?